### PR TITLE
Update date-converter extension

### PR DESCRIPTION
**Pull request by vineus**
## Description

* add 24h time format option (disabled by default)

## Screencast

image

Raycast 2024-05-19 at 14 04 28

image

Raycast 2024-05-19 at 14 04 56

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by asportnoy — CHANGES_REQUESTED** [review]
Could the relative info be part of the item description maybe? Also does it need to be that detailed?

**Comment by vineus** [comment]
@asportnoy 
> Could the relative info be part of the item description maybe? Also does it need to be that detailed?

Better?

date-converter 2024-05-19 at 22 50 48

**Comment by asportnoy** [comment]
- I think the relative time is still too detailed. "9 minutes" should be plenty detailed for this, I think the seconds is overkill
- Can you add a prefix or suffix like "in/ago"? So "9 minutes ago"
- Separate the two parts with something like a hyphen and ditch the "relative:"

**Review by asportnoy — COMMENTED** [review]
Marking as reviewed again - see above

**Comment by vineus** [comment]
Reverted the relative time work.

I'll use my fork, I guess my needs are very specific.

**Comment by pernielsentikaer** [comment]
We can make it a preferences to show seconds or not, I like the idea 🙂

**Comment by vineus** [comment]
Found some time to update.

Here is a shorter version:

image

image

image

**Comment by pernielsentikaer** [comment]
@asportnoy could you check it once again

File: extensions/date-converter/src/index.tsx [hunk 1]
```diff
@@ -183,7 +190,7 @@ export default function Command() {
         <List.Item
           key={date.toISOString()}
           title={humanFormatter.format(date)}
-          subtitle={label}
+          subtitle={label + "﹣" + timeAgo.format(date)}
```
Comment (asportnoy) [inline extensions/date-converter/src/index.tsx:193]:
```suggestion
          subtitle={`${label} - ${timeAgo.format(date)}`}
```

**Review by asportnoy — COMMENTED** [review]
Much better! One minor thing and then I think it's good to go.

**Review by asportnoy — APPROVED** [review]
Thanks for those changes, looks good to me.

@pernielsentikaer ready to merge when you are.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**